### ISSUE TITLE
HSTU的复现中，labels都是0

### ISSUE THREAD

**Issue by bityigoss** [bytedance/HLLM#11]
https://github.com/bytedance/HLLM/blob/2deed176960901aa88a8864e6f6902a194c4e970/code/REC/model/IDNet/hstu.py#L515C1-L516C1
是否应该增加
`labels[0] = 1`

**Comment by ssyzeChen** [comment]
Hi，NCELoss中正样本 idx=0

**Comment by bityigoss** [comment]
> Hi，NCELoss中正样本 idx=0
@ssyzeChen 
hi, 正因为正样本index=0，那么labels[0]应该是1吧，代码中labels初始化之后都是0

**Comment by ssyzeChen** [comment]
> > Hi，NCELoss中正样本 idx=0
> > @ssyzeChen
> > hi, 正因为正样本index=0，那么labels[0]应该是1吧，代码中labels初始化之后都是0

抱歉我没太理解这个问题，预测的目标是正样本，因此所有label才是0
在 [bs, samples] 的pred_logits中，idx=0是正样本，其余为负样本

**Comment by laymanColton** [comment]
请问nce_loss代码中是否需要增加
labels[0] = 1
